.github/scripts/license_checker.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+
+busl_files=$(grep -r 'SPDX-License-Identifier: BUSL' --exclude=./.github/scripts/license_checker.sh .)
+
+# If we do not find a file in .changelog/, we fail the check
+if [ -n "$busl_files" ]; then
+    echo "Found BUSL occurrences in the PR branch!"
```

Comment:
Can you name drop the JIRA ticket (NET-5258) in this message so devs who run into this can understand why?

Reply:
Good idea, done